### ISSUE TITLE
There is no announcement for screenreaders that Save / Update Page action was successful

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#95513]
## Issue Description
This issue was identified in the [accessibility audit of the review & confirmation pages ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94614)of the 526 form.

[**Expected severity**](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric): a11y-defect-1
**Page**: review-and-submit
**Environment**: any screenreader/browser combo
**Estimated Level of Effort to fix**: Medium
**Design system component involved**: Yes
**Relevant WCAG success criteria**: [4.1.3 Status Messages Level AA](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_overview#status-messages)

### To Reproduce:
- Use a screenreader and edit an item under one of the review accordions. 
- Click Save or Update Page. 

#### Expected Behavior
-  There should be an announcement for the screenreader user that item has been updated/saved.

#### Actual Behavior
- There is no announcement that the change was successful.

## Tasks
- [ ] Add an aria-live polite area to announce this change. It should not steal focus.
- [ ] Please update the "Fix Status" column in the [526 Accessibility Audit issues tracking doc](https://docs.google.com/spreadsheets/d/1paNxD798NSMrjZeUe9WwVQ4xRdl439ypKyPDPW3CXIo/edit?gid=2034611899#gid=2034611899) when ticket is started and completed.

## Acceptance Criteria
- [ ] Screenreader users hear that their change was successful.
- [ ] Accessibility Audit issue tracking doc is updated with ticket status
